app/views/users/sessions/email_not_found.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l">You do not have an account</h2>
-    <p>Your school may not be registered. </p>
-    <%= govuk_link_to "Register your school", users_check_email_path %>
+    <h2 class="govuk-heading-l">Your school has not registered</h2>
+    <p>You must be responsible for running your induction programme to register your school.</p>
+    <%= govuk_link_to "Register your school", users_information_path, button: true %>
```

Comment:
Is this not meant to be `users_new_path`?

Reply:
No, first to those information pages you had in your screenshots earlier

Reply:
Are you working from these designs? 
![image](https://user-images.githubusercontent.com/40488007/106033934-3132b880-60ca-11eb-9e20-bc0dc94cd123.png)

Reply:
Which design are you working from?

Reply:
https://miro.com/app/board/o9J_lZ2k8oc=/

Reply:
Al and Shani have been updating it so it's slightly diff from what it was. I'll have to update it later when they are done.